indico/modules/rb/client/js/components/TimeRangePicker.jsx

```diff
@@ -26,9 +26,11 @@ function _humanizeDuration(duration) {
   if (hours === 1 && minutes === 0) {
     return Translate.string('1 hour');
   } else if (hours !== 0) {
-    return Translate.string('{time} hours', {time: hours + minutes / 60});
+    return PluralTranslate.string('{time} hour', '{time} hours', hours + minutes / 60, {
+      time: hours + minutes / 60,
```

Comment:
I'd also go for a more appropriate name for the placeholder, like `count` maybe?
```suggestion
    return PluralTranslate.string('{count} hour', '{count} hours', hours + minutes / 60, {
      count: hours + minutes / 60,
```